README.md

```diff
@@ -0,0 +1,7 @@
+# What is ChatKC
+
+ChatKC is a specialized WSS (**W**eb**S**ocket **S**ecure)  chat created by the popular youtuber, [MattKC](https://www.youtube.com/mattkc).<br><br>
+ChatKC is different because it uses Google oAuth to authenticate users to the server after sign in.<br><br>
+After the user is authenticated, they may chat on the MattKC site, or on an alternative, unofficial specialized client.<br><br>
+Work is being done to fully reverse-engineer the ChatKC usage, so that others can make a decentralized network of ChatKC-style servers.<br><br>
+
```

Comment:
This is already covered in `protocol.md`. Maybe we should rename the file to `readme.md` so it's easier to find?